src/sagemaker/huggingface/training_compiler/config.py

```diff
@@ -102,6 +102,17 @@ def validate(cls, estimator):
 
         super(TrainingCompilerConfig, cls).validate(estimator)
 
+        if estimator.pytorch_version:
+            if (Version(estimator.pytorch_version) in SpecifierSet("< 1.9")) or (
+                Version(estimator.pytorch_version) in SpecifierSet("> 1.11")
+            ):
+                error_helper_string = (
+                    "Training Compiler is only valid between HuggingFace PyTorch 1.9-1.11 "
```

Comment:
Training Compiler is only valid between -> SageMaker Training Compiler is only supported with